Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -366,6 +366,12 @@ public enum DefaultFunctionOpacityOptions {
 `opaque` means functions are always opaque so the opaque keyword is not needed, and functions must be revealed everywhere needed for a proof.".TrimStart()) {
   };
 
+  public static readonly Option<bool> TranslateStandardLibrary = new("--translate-standard-library", () => true,
+    @"Currently Dafny does not prevent separately built Dafny projects from each including the standard library,
+leading to conflicts. When combining such projects, please ensure that only one of them has --translate-standard-library set to true.
```

Comment:
The first sentence feels overly defensive to me, but it might be because we see the current state differently. I don't think it's inherently wrong to merge doo files personally, building larger aggregate libraries, as long as you don't use them incorrectly.

Reply:
> I don't think it's inherently wrong to merge doo files personally, building larger aggregate libraries, as long as you don't use them incorrectly.

What do you mean by merging library files?

Reply:
As in say you have an `A.doo` that depends on a `B.doo` and a `C.doo` library. I could see it being useful to call `dafny translate -t:lib A.doo B.doo C.doo --output D` to create an aggregate, self-contained `D.doo` library, especially while we don't have fleshed-out distribution of Dafny libraries with dependency support.

But this isn't important to agree on right now, your rewording makes me happy.